src/main/java/core/basesyntax/enums/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.enums;

```

Comment:
do not use plural in package names
```suggestion
package core.basesyntax.enum;

```

Reply:
The same situation with "interfaces" package: "interface" - is not allowed. Renamed as "interfaceforfigure".